.github/workflows/iroha2-dev-code-quality.yml

```diff
@@ -2,11 +2,11 @@ name: I2::Dev::CodeQuality
 
 on:
   workflow_run:
-    workflows: ["I2::Dev::Tests", "I2::Dev::Static"]
```

Comment:
Might help to reduce creating duplicated runs. Also trigger only if parent workflow was started to be executed.

Reply:
makes senes